src/Colourista/Pure.hs

```diff
@@ -214,3 +218,24 @@ reset = fromString $ setSGRCode [Reset]
 {-# SPECIALIZE reset :: String     #-}
 {-# SPECIALIZE reset :: Text       #-}
 {-# SPECIALIZE reset :: ByteString #-}
+
+-- | Code to apply __singleUnderline__ emphasis for the terminal output.
+singleUnderline :: IsString str => str
+singleUnderline = fromString $ setSGRCode [SetUnderlining SingleUnderline]
+{-# SPECIALIZE singleUnderline :: String     #-}
+{-# SPECIALIZE singleUnderline :: Text       #-}
+{-# SPECIALIZE singleUnderline :: ByteString #-}
+
+-- | Code to apply __doubleUnderline__ emphasis for the terminal output.
```

Comment:
I have checked `ansi-terminal` docs, and it states that double underline doesn't work for Windows. Could you please also add this to the docs in here? 🙂

Reply:
Sure, I've added that.